core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -231,14 +232,16 @@ private synchronized void initContext(ClientContext ctx,
         .setMasterInquireClient(masterInquireClient).build();
     mMetricsEnabled = getClusterConf().getBoolean(PropertyKey.USER_METRICS_COLLECTION_ENABLED);
     if (mMetricsEnabled) {
-      try {
-        InetSocketAddress masterAddr = masterInquireClient.getPrimaryRpcAddress();
-        mMasterClientContext.loadConf(masterAddr, true, true);
-      } catch (UnavailableException e) {
-        LOG.error("Failed to get master address during initialization", e);
-      } catch (AlluxioStatusException ae) {
-        LOG.error("Failed to load configuration from "
-            + "meta master during initialization", ae);
+      if (CommonUtils.PROCESS_TYPE.get() != CommonUtils.ProcessType.MASTER) {
```

Comment:
Does it make sense to report client metrics at all from the master?

Reply:
Yeah, it makes sense to not report client metrics at all from the master. If not reporting, we may lose some of the client metrics, but client metrics in the master process is wrong no matter what. This is because of the client metrics in master may record as `Master.Client.metricName` which will not be reported. Will add a TODO to solve master client metrics in the future.